kitchensink/src/app/common/PageElements/FoundationElements/ColorRow/colorRow.css

```diff
@@ -0,0 +1,64 @@
+.colorRowContainer{
+  margin: 24px 0; 
+}
+
+.colorRowContainer table {
+  width: 100%;
+  table-layout: fixed;
+
+  & .title {
+    height: 14px;
+    font-weight: 500;
+    text-transform: uppercase;
+    color: var(--neutral-200);
+
+    &.large {
+      font-size: 18px;
+    }
+
+    &.medium{
+      font-size: 16px;
+    }
+
+    &.small {
+      font-size: 12px;
+    }    
+  }   
+
+  & thead th {
+    height: 13px;
+  }
+
+  & thead tr .base-column {
+    text-align: left;
+  }  
+}
+
+.colorRow {
+  background: #FFFFFF;
+  box-shadow: inset 0px 1px 0px #E1E3E8, inset 0px -1px 0px #E1E3E8;
```

Comment:
could you please reuse the variable  ` neutral050: '#e1e3e8',`

located here : `/Users/thr/Projects/pyrene/pyrene/src/styles/colorConstants.ts`